### PR TITLE
User can sign in

### PR DESCRIPTION
**Pull request by SharjeelSIqbal**
https://user-images.githubusercontent.com/50930123/141598967-0de592d3-ee6a-4ea0-8f7c-ddc7e0cf6f10.mov

### CODE REVIEW

File: utils/database.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { MongoClient } from 'mongodb';
```
Comment (humbertovnavarro) [inline utils/database.js:1]: Are we able to make this  file typescript?

**Review by humbertovnavarro — APPROVED** [review]
Approved, but see comments.

File: utils/database.js (hunk 1 above)
Comment (SharjeelSIqbal) [reply utils/database.js:1]: I tried. If you can do it or know how to do it make it a typescript file but I was confused and looked into but couldn't find anything

Comment (humbertovnavarro) [reply utils/database.js:1]: not a big deal

**Pull request merged**